examples/adjoints/adjoints_ex6/element_postprocess.C

```diff
@@ -0,0 +1,62 @@
+// General libMesh includes
+#include "libmesh/libmesh_common.h"
+#include "libmesh/elem.h"
+#include "libmesh/fe_base.h"
+#include "libmesh/fem_context.h"
+#include "libmesh/point.h"
+#include "libmesh/quadrature.h"
+
+// Local includes
+#include "poisson.h"
+
+// Bring in everything from the libMesh namespace
+using namespace libMesh;
+
+// Last modified: Feb 13, 2009
+
+// Define the postprocess function to compute QoI 0, the integral of the the solution
+// over a subdomain
+
+void PoissonSystem::element_postprocess (DiffContext &context)
+
+{
+  FEMContext &c = cast_ref<FEMContext&>(context);
```

Comment:
This and several other variable declarations don't follow the formatting guidelines with regard to [spacing](https://github.com/libMesh/libmesh/wiki/Source-code-formatting-conventions#spacing-and-indentation). You can run contrib/bin/operator_spacing.pl on any file and it will print lines with "suspicious" formatting that you can go fix.